docs/docs/benchmarks-introduction.mdx

```diff
@@ -0,0 +1,132 @@
+---
+id: benchmarks-introduction
+title: Introduction
+sidebar_label: Introduction
+---
+
+## Quick Summary
+
+In `deepeval`, benchmarks provide a standardized method to evaluate LLM performance. Each benchmark assesses various aspects of language model performance. `deepeval` offers a range of popular benchmarks for you to quickly get started with, such as:
+
+- BIG-Bench Hard
+- HellaSwag
+- MMLU (Massive Multitask Language Understanding)
+
+To assess your LLM's performance on a benchmark, you will need to create a model with `DeepEvalLLMBase`. More information on this can be found [here](./metrics-introduction.mdx).
+
+## Benchmark Evaluation
+
+Below is an example of how to evaluate the Mistral 7B model against the `MMLU` benchmark. Start by creating a custom model with `DeepEvalLLMBase`.
+
+```python
+from deepeval.benchmarks.base_benchmark import DeepEvalBaseLLM
+from transformers import AutoTokenizer, AutoModelForCausalLM
+
+class Mistral7B(DeepEvalBaseLLM):
+    def __init__(self, model):
+        self.model = model
+        self.tokenizer = AutoTokenizer.from_pretrained(
+            "mistralai/Mistral-7B-v0.1"
+        )
+
+    def load_model(self):
+        return self.model
+
+    def _call(self, prompt: str) -> str:
```

Comment:
flagging this, change to generate and a_generate. For a_generate for models through the hugging face interface I'm sure there is an async version, if not just use the same implementation as generate (although I'm hoping there is)

Reply:
I couldn't find the async generate version from HF docs TT